### DISCUSSION TITLE
Having issues with accessing System API

### DISCUSSION THREAD

**Discussion by huzaifaali14** [zitadel/zitadel#4668] [Q&A] [answered]
I am trying to access the system API and followed [this](https://docs.zitadel.com/docs/guides/integrate/access-zitadel-system-api) guide for setting up the user and creating access token but unfortunately I am hitting this error `parsing of request failed` which is not really clear whats the actual problem.

<img width="1050" alt="Screenshot 2022-11-07 at 10 21 26 AM" src="https://user-images.githubusercontent.com/30002668/200232171-16117107-560e-4ebe-a964-01e916e57f34.png">

**Comment by livio-a** [comment, marked as answer]
Hey @huzaifaali14 

At a first glance, everything seems ok. Is the body empty? And could you send me the headers you've set?
If there's anything confidential you can also send us an email to support@zitadel.com

**Reply by huzaifaali14** [reply]
Hey @livio-a 

This is the request 
`curl --request POST \
  --url http://localhost:8080/system/v1/instances/_search \
  --header 'Authorization: Bearer eyJhbGciOiJSUzI1NiJ9.eyJpc3MiOiJzeXN0ZW0tdXNlci0xIiwic3ViIjoic3lzdGVtLXVzZXItMSIsImF1ZCI6WyJodHRwOi8vbG9jYWxob3N0OjgwODAiXSwiaWF0IjoxNjY3ODA4Njc0LCJleHAiOjE2Njc4MTIyNzR9.H1nlrwd10AdVjZt6lzfLPztLhlhkEmvZ7dJTsupNeXPs-8JCyGRJeLK6fWkz0EEemv352XeeXq3HyB43AOmjuiJ8KwXd88Rq0gn6DqyMclCbGOIal4LE1vNnaW-Z47i-ZKeTLfDZMNKkp0N7fmIm9mF0fHWkQLPpa0XFrRrGospQNYuqqJwyQGLTBwneUPNnBZ0F28AWb2m99O3b27YiTTfSGLglFpa7NPsakFUd7vsscPYfhazV0TVL5H4KjpNymUZclORtUyyu5xstIjabWS1sYPfY00R13NW4ViP4fu5wldjnPzMf6U33YwXJfHlv0o6K9Y46dpX9411hPuYxjA%' \
  --header 'Content-Type: application/json'`

**Reply by huzaifaali14** [reply]
<img width="1047" alt="Screenshot 2022-11-07 at 2 56 58 PM" src="https://user-images.githubusercontent.com/30002668/200281483-fa25acc2-ae79-4bb5-ba97-92e94d17cede.png">

**Reply by livio-a** [reply]
I assume you used zitadel-tools to generate the JWT, right? We've found an issue in the output of the token.
We currently print the JWT without a newline, which may result (e.g. in zsh) in a printed EOL character (`%`).
We already created a PR https://github.com/zitadel/zitadel-tools/pull/59 for this and will be released today.
In the meantime just omit the `%` at the end.

**Reply by huzaifaali14** [reply]
Hey @livio-a yeah that was the issue. Thank you its working now.
